### PR TITLE
✨ Here is Hurry

### LINKED ISSUE #67: Implement `mgm-hurry`
Implement a `mgm-manager`, which will:
- Simplify and shorten the usage of the go-to commands + remove the requirement to copy/paste them
- Possibly automate more of the MGM optimization process by queuing HyperOpts after eachother *(This will only be required if we find a new HyperOpt process that requires this, looking at https://github.com/Rikj000/MoniGoMani/issues/51 and https://github.com/Rikj000/MoniGoMani/issues/47 for this)*

I'm planning to create the `mgm-manager` out of [Hurry](https://github.com/Rikj000/MoniGoMani/pull/60) and [Tool Belt](https://github.com/Rikj000/MoniGoMani/pull/66) as the base.
Already want to thank @denvers and @rvasilev for their great work on both their pending PRs, they will definitely be used!
But I hope you guys understand that I want to wrap up the last issues from `v0.12.0` first so we can finally release the thing! :rocket:

### PR DESCRIPTION
**Pull request by denvers**
Here is the CLI tool I use to make things a bit easier. It's called: Hurry. Because it enables me to use freqtrade/mgm a lot faster.

```
hurry – 💨 Your command is my objective 💨

Install dependencies:
$ pip install fire logger pyaml

Usage:
$ python3 hurry --help
$ python3 hurry [COMMAND] --help

$ python3 hurry setup
$ python3 hurry cleanup
$ python3 hurry download_candle_data
$ python3 hurry download_static_pairlist
$ python3 hurry hyperopt
$ python3 hurry hyperopt_show_results {epoch}
$ python3 hurry hyperopt_apply_epoch {epoch}

All in one a.k.a. Show me the money:
$ python3 hurry cleanup \
                download_candle_data \
                download_static_pairlist \
                hyperopt \
                hyperopt_show_results 1 \
                hyperopt_apply_epoch 1
```

### CODE REVIEW

**Comment by denvers** [comment]
Install dependencies:
`$ pip3 install art fire logger pyaml PyInquirer`

Usage:
```$ python3 hurry --help
$ python3 hurry [COMMAND] --help

$ python3 hurry setup

$ python3 hurry cleanup
$ python3 hurry download_candle_data
$ python3 hurry download_static_pairlist
$ python3 hurry hyperopt
$ python3 hurry hyperopt_show_results {epoch}
$ python3 hurry hyperopt_apply_epoch {epoch}
```

Setup example:

```
$ python3 hurry setup
 _                                       _  _
| |__   _   _  _ __  _ __  _   _    ___ | |(_)
| '_ \ | | | || '__|| '__|| | | |  / __|| || |
| | | || |_| || |   | |   | |_| | | (__ | || |
|_| |_| \__,_||_|   |_|    \__, |  \___||_||_|
                           |___/

 >> Freqtrade binary: `docker-compose run --rm freqtrade`
 + Let's save some answers to make our lifes easier

? Which way you want to use Freqtrade?  docker
? Please enter the default timerange you want to use  20201201-20210316
? Which HyperOpt Strategy do you want to use?  MoniGoManiHyperStrategy
? Which HyperOpt Loss do you want to use?  WinRatioAndProfitRatioLoss
? Which spaces do you want to HyperOpt?  [all]
? Please enter the default quotation you want to use  USDT
? Please enter the amount of epochs you want to HyperOpt  100
? Which exchange do you want to use?  binance
? Please enter the exchange API key  ****************************************************************
? Please enter the exchange API secret  ****************************************************************
 >> Configuration data written to .hurry

 >> Exchange settings written to mgm-config-private.json
```

**Comment by rucin93** [comment]
I don't recommend using PyInquirer package, because it's not maintained anymore. In my CLI i'm using similar project - InquirerPy :)

File: hurry [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    """
+    Saves setup data based on your answers
+    """
+    print(" + Let's save some answers to make our lifes easier\n")
+
+    questions = [
+      {
+        'type': 'list',
+        'name': 'install_type',
+        'message': 'Which way you want to use Freqtrade?',
```
Comment (muri72) [inline hurry:108]: what about conda ?

**Review by muri72 — COMMENTED** [review]
Support for conda?

**Comment by rucin93** [comment]
> Support for conda?

Screw conda :)

**Comment by topscoder** [comment]
Most actual version of Hurry is now available here: https://github.com/topscoder/hurry-cli

**Comment by topscoder** [comment]
> I don't recommend using PyInquirer package, because it's not maintained anymore. In my CLI i'm using similar project - InquirerPy :)

Thank you. Switched to InquirerPy 💪

**Comment by topscoder** [comment]
Way too much hours invested in Hurry. Couldn't let it go. But.... the good part is that there are many new features.  💪

Docs are up to date. I'd invite you to take a look at: https://github.com/topscoder/hurry-cli

Please feel free to provide additional requirements or 'nice-to-haves'.

**Comment by Rikj000** [comment]
> Way too much hours invested in Hurry. Couldn't let it go. But.... the good part is that there are many new features. muscle

No worries man :smile: It's a great stand alone tool for many strategies!

**Pull request closed without merging**